main/src/hadd.cxx

```diff
@@ -483,7 +483,7 @@ int main( int argc, char **argv )
       } else {
          if (!keepCompressionAsIs && merger.HasCompressionChange()) {
             // Don't warn if the user any request re-optimization.
```

Comment:
Maybe not for this PR, but this comment has surely at least a typo, possibly the comment is misplaced altogether. @pcanal what do you think?

Reply:
It's a typo, it means that the warning is not given if the user explicitly asked for reoptimization (at least, that's how I interpret it). I'll fix the typo, thanks.